src/backups/backup.ts

```diff
@@ -405,21 +444,27 @@ export class Backup implements Responsable {
      * Builds mysqldump/mysql command
      */
     buildCmd(cmd: 'mysqldump' | 'mysql', databaseService: DatabaseService): string {
+        const config = app().config
         const dbConfig: DatabaseConfig = databaseService.config;
-        const dumpFile = path.join(this._backupPath, Backup.DUMP_FILENAME);
+        let dumpFile:string = path.join(this._backupPath, Backup.DUMP_FILENAME);
 
+        if( cmd === 'mysql') {
+            fs.existsSync(path.join(this._backupPath, Backup.DUMP_FILENAME)) ? dumpFile =  path.join(this._backupPath, Backup.DUMP_FILENAME)
+        : dumpFile = path.join(config.get('tmp.directory'), path.basename(this._backupPath), Backup.DUMP_FILENAME);
+         
+        }
+        
         const shellescape = require('shell-escape');
         process.env.MYSQL_PWD = shellescape([dbConfig.pass]).substring(0,128);
 
         const dir = cmd==='mysqldump' ? '>' : '<';
-
+        const statistics = cmd==='mysqldump' ?'--column-statistics=0 ' : ''
```

Comment:
@cmunyoz adding this flag to the `mysqldump` looks like mandatory when `mysqldump` version is > 8.0 (https://stackoverflow.com/questions/52423595/mysqldump-couldnt-execute-unknown-table-column-statistics-in-information-sc) . 

Do you think adding this flag could have some kind of unexpected behaviour?

Reply:
Our CI is setting this parameter using the configuration file. Does that mean that we should not set any flag when call to `mysqldump` and leave the responsibility to the end user?